src/js/components/Grid/stories/Percentages.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 
-import { Grommet, Box, Grid } from 'grommet';
-import { grommet } from 'grommet/themes';
+import { Box, Grid } from 'grommet';
 
 export const Percentages = () => (
-  <Grommet theme={grommet} full>
+  // Uncomment <Grommet> lines when using outside of storybook
+  // <Grommet theme={...}>
+  <Box height="medium">
```

Comment:
I need to set a height to Grid be visible. Let me know what is the best option to use Box with height=100vh or similar, please.

Reply:
We can set the full prop to true for this story (see comment below)